proto/src/determined/api/v1/api.proto

```diff
@@ -38,7 +38,7 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
     schemes: [HTTP, HTTPS],
     external_docs: {
         description: "Determined AI Documentation",
-        url: "https://docs.determined.ai/",
+        url: "/docs",
```

Comment:
so far the user can access swagger-ui only if they have an active cluster in which case it'd be better to direct them to their local docs as opposed to the hosted version.